lib/oeqa/selftest/cases/updater_qemux86_64.py

```diff
@@ -379,7 +369,7 @@ def is_ecu_registered(self, ecu_id):
             return not_reg_start == -1 or (device_status.find(ecu_id[1], not_reg_start) == -1)
 
         def get_info(self):
-            stdout, stderr, retcode = self.send_command('aktualizr-info')
+            stdout, stderr, retcode = self.send_command('aktualizr-info --wait-until-provisioned', timeout=620)
```

Comment:
`620` seems like an arbitrary number, and perhaps a bit high. Any particular reason for that choice? Do you think it needs to be that long?

Reply:
I think, this number has to be higher than this one https://github.com/advancedtelematic/meta-updater/blob/9b397b1e047c23921e07258b8a1664d55d850488/recipes-test/demo-config/shared-conf.inc#L5 which is equal to 240 currently.

Reply:
Yes, agreed, but almost three times as long is probably more than necessary. But it doesn't really matter, and let's just go with what you have until we have a good reason to change it. 10 minutes isn't too bad, anyway.